autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/ui/credential/saving/AutofillSavingCredentialsViewModel.kt

```diff
@@ -31,4 +34,45 @@ class AutofillSavingCredentialsViewModel @Inject constructor() : ViewModel() {
     fun showOnboarding(): Boolean {
         return autofillStore.showOnboardingWhenOfferingToSaveLogin
     }
+
+    fun determineTextResources(credentials: LoginCredentials): DisplayStringResourceIds {
```

Comment:
NIT: This could be part of the viewmodel's view state or maybe the LoginCredentials could be kept in the ViewModel rather than the fragment.

Reply:
I agree it'd be better wiring it in through the view state, and that's likely what we'll end up doing anyway, but wanted to minimise changes on that right now.

I'm going to defer such further changes to another PR.